posthog/models.py

```diff
@@ -121,6 +121,7 @@ def create_with_data(self, users: List[User]=None, **kwargs):
             type='ActionsTable',
             filters={TREND_FILTER_TYPE_ACTIONS: [{'id': action.pk, 'type': TREND_FILTER_TYPE_ACTIONS}], 'display': 'ActionsTable', 'breakdown': '$browser'}
         )
+        DashboardItem.objects.create(team=team, name='Unique Pageviews this week (DAU)', type='ActionsLineGraph', filters={TREND_FILTER_TYPE_ACTIONS: [{'id': action.pk, 'math': 'dau', 'type': TREND_FILTER_TYPE_ACTIONS}]})
```

Comment:
I think maybe "Daily Active Users" would be better. DAU is surprisingly not well known.